### PR TITLE
Feature/Add user picture to nav drawer

### PR DESCRIPTION
**Pull request by friedger**
This PR adds the user picture to the nav drawer when logged in.
It is a basic implementation and does not cover edge cases.
When not logged in a grey avatar is shown.

device-2015-04-29-102752

device-2015-04-28-234557

### CODE REVIEW

**Comment by friedger** [comment]
This is the last part for #234

**Comment by stefanhoth** [comment]
That looks great!

Could you add a screenshot for the logged out state?

File: app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java [hunk 1]
```diff
@@ -249,10 +259,22 @@ protected void closeNavDrawer() {
         }
     }
 
+    protected void updateUserPicture() {
+        com.google.android.gms.plus.model.people.Person user = com.google.android.gms.plus.Plus.PeopleApi.getCurrentPerson(getGoogleApiClient());
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java]: Why the full package name?

**Comment by tasomaniac** [comment]
It is a little bit detail but the icons of the list items and the user profile logo are left aligned in Google's implementation.

And we can add users name and email just like in the Google's implementation without a Account switcher Spinner. 

And since we have lots of items in the Nav Drawer, I think we should split them right? That can be another issue though.

File: app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java (hunk 1 above)
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java]: we are also using the com.google..api... classes

**Comment by friedger** [comment]
user name and email might be difficult to read depending on the chapter image. For the profile we have always a white border.

**Comment by friedger** [comment]
Screenshots updated

**Comment by tasomaniac** [comment]
@friedger then it is a good reason to use new fancy ColorUtils class from support library to find contrasting text color :)
https://chris.banes.me/2015/04/22/support-libraries-v22-1-0/

**Comment by friedger** [comment]
@tasomaniac I think that is too much for this release. Finding the right color of the area below the text, etc...

**Comment by friedger** [comment]
added #338

**Comment by tasomaniac** [comment]
I was just flying on the clouds. (A Turkish idiom for suggesting not urgent and hard things) :smile: 

BTW, I think we can remove not-logged-in avatar completely for now and merge this. The avatar will be a great improvement but it looks ugly when it is not rounded.

**Comment by friedger** [comment]
Screenshot updated.

**Comment by tasomaniac** [comment]
I think it is really nice. ty

**Pull request merged**